protocol/protocol.go

```diff
@@ -37,22 +37,22 @@ var (
 	ERROR_COMMAND_PARSE = errors.New("Command parse error")
 
 	//Commands declared once for convenience
-	DEL_COMMAND         = []byte{'d', 'e', 'l'}
-	SUBSCRIBE_COMMAND   = []byte{'s', 'u', 'b', 's', 'c', 'r', 'i', 'b', 'e'}
-	UNSUBSCRIBE_COMMAND = []byte{'u', 'n', 's', 'u', 'b', 's', 'c', 'r', 'i', 'b', 'e'}
-	PING_COMMAND        = []byte{'p', 'i', 'n', 'g'}
-	INFO_COMMAND        = []byte{'i', 'n', 'f', 'o'}
-	SHORT_PING_COMMAND  = []byte{'P', 'I', 'N', 'G'}
-	SELECT_COMMAND      = []byte{'s', 'e', 'l', 'e', 'c', 't'}
-	QUIT_COMMAND        = []byte{'q', 'u', 'i', 't'}
+	DEL_COMMAND         = []byte("del")
+	SUBSCRIBE_COMMAND   = []byte("subscribe")
+	UNSUBSCRIBE_COMMAND = []byte("unsubscribe")
+	PING_COMMAND        = []byte("ping")
+	INFO_COMMAND        = []byte("info")
+	SHORT_PING_COMMAND  = []byte("PING")
+	SELECT_COMMAND      = []byte("select")
+	QUIT_COMMAND        = []byte("quit")
```

Comment:
I think this does conversion first to bytes before creating byte array. Isn't it good idea to leave them as byte array especially if protocol is being called repetitively.